pkg/cli/cmd/cli.go

```diff
@@ -190,6 +240,7 @@ func init() {
 	rootCmd.Flags().IntP("timeout", "t", 5, "timeout duration in seconds")
 	rootCmd.Flags().BoolP("lineend", "l", true, "line end")
 	rootCmd.Flags().BoolP("verbose", "v", false, "verbose mode")
+	rootCmd.Flags().BoolP("urc", "u", false, "unsolicited response code")
```

Comment:
I recommend informing the user that; if urc flag is set, the command will not be sent to modem.
```suggestion
	rootCmd.Flags().BoolP("urc", "u", false, "unsolicited response code - if this flag is set, command will not be sent to modem")
```

<br>

Correct me if I'm wrong but this is what I've understand from the following line:

https://github.com/sixfab/atcomv2/blob/137248c8736c28ff5106ecf9f6c57a26da7dcc24/atcom.go#L130-L132